packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -63,11 +64,11 @@ class PreferencesController {
 }
 
 class InspectorPreferencesController {
-  final ValueNotifier<bool> _hoverEvalMode = ValueNotifier(false);
-  final String _hoverEvalModeStorageId = 'inspector.hoverEvalMode';
-
   ValueListenable<bool> get hoverEvalModeEnabled => _hoverEvalMode;
 
+  final ValueNotifier<bool> _hoverEvalMode = ValueNotifier(false);
+  static const String _hoverEvalModeStorageId = 'inspector.hoverEvalMode';
```

Comment:
remove type on LHS.  For the ValueNotifier, you can move the type to the RHS: `ValueNotifier<bool>(false);`